source/docs/software/advanced-controls/trajectories/ltv.rst

```diff
@@ -0,0 +1,77 @@
+   ```
+
+   ```c++
+   frc::LTVUnicycleController controller{{0.0625, 0.125, 2.0}, {1.0, 2.0}, 0.02_s, 9_mps};
+   ```
+
+   ```python
+   controller = LTVUnicycleController([0.0625, 0.125, 2.0], [1.0, 2.0], 0.02, 9)
+   ```
+
+## Getting Adjusted Velocities
+The LTV Unicycle controller returns "adjusted velocities" so that the when the robot tracks these velocities, it accurately reaches the goal point. The controller should be updated periodically with the new goal. The goal comprises of a desired pose, desired linear velocity, and desired angular velocity. Furthermore, the current position of the robot should also be updated periodically. The controller uses these four arguments to return the adjusted linear and angular velocity. Users should command their robot to these linear and angular velocities to achieve optimal trajectory tracking.
+
+.. note:: The "goal pose" represents the position that the robot should be at a particular timestep when tracking the trajectory. It does NOT represent the final endpoint of the trajectory.
+
+The controller can be updated using the ``Calculate`` (C++) / ``calculate`` (Java/Python) method. There are two overloads for this method. Both of these overloads accept the current robot position as the first parameter. For the other parameters, one of these overloads takes in the goal as three separate parameters (pose, linear velocity, and angular velocity) whereas the other overload accepts a ``Trajectory.State`` object, which contains information about the goal pose. For its ease, users should use the latter method when tracking trajectories.
+
+.. tab-set-code::
+
+   ```java
+   Trajectory.State goal = trajectory.sample(3.4); // sample the trajectory at 3.4 seconds from the beginning
+   ChassisSpeeds adjustedSpeeds = controller.calculate(currentRobotPose, goal);
+   ```
+
+   ```c++
+   const Trajectory::State goal = trajectory.Sample(3.4_s); // sample the trajectory at 3.4 seconds from the beginning
+   ChassisSpeeds adjustedSpeeds = controller.Calculate(currentRobotPose, goal);
+   ```
+
+   ```python
+   goal = trajectory.sample(3.4)  # sample the trajectory at 3.4 seconds from the beginning
+   adjustedSpeeds = controller.calculate(currentRobotPose, goal)
+   ```
+
+These calculations should be performed at every loop iteration, with an updated robot position and goal.
+
+## Using the Adjusted Velocities
+The adjusted velocities are of type ``ChassisSpeeds``, which contains a ``vx`` (linear velocity in the forward direction), a ``vy`` (linear velocity in the sideways direction), and an ``omega`` (angular velocity around the center of the robot frame).
+
+The returned adjusted speeds can be converted to usable speeds using the kinematics classes for your drivetrain type. For example, the adjusted velocities can be converted to left and right velocities for a differential drive using a ``DifferentialDriveKinematics`` object.
```

Comment:
We should mention earlier in the article that this controller only applies to nonholonomic drivetrains like differential drive. Holonomic drivetrains can use more optimal controllers like separate decoupled PD controllers, because they don't have to deal with nonholonomic limitations like not being able to move sideways.